fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -83,13 +83,14 @@ async def get_fee_4(address: str, contract: Contract, factory_contract: Contract
     """
     return await factory_contract.caller.getPairFee(address, await contract.caller.stable())
 
+
 EXCHANGE_INFO = {
     "velocimeter_v2": {"decimals": 4, "factory_abi": VELOCIMETER_V2_FACTORY_ABI, "pool_abi": VELOCIMETER_V2_POOL_ABI, "fee_function": get_fee_1},
     "equalizer_v2": {"decimals": 4, "factory_abi": SCALE_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_2},
     "aerodrome_v2": {"decimals": 4, "factory_abi": SOLIDLY_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_3},
     "velodrome_v2": {"decimals": 4, "factory_abi": SOLIDLY_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_3},
     "scale_v2": {"decimals": 18, "factory_abi": SCALE_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_2},
-    "cleopatra_v2": {"decimals": 4, "factory_abi": CLEOPATRA_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_4},
+    "cleopatra_v2": {"decimals": 18, "factory_abi": CLEOPATRA_V2_FACTORY_ABI, "pool_abi": CLEOPATRA_V2_POOL_ABI, "fee_function": get_fee_4},
```

Comment:
"decimals": 18 is wrong, the current value 4 is correct.

Reply:
https://explorer.mantle.xyz/address/0xba94D26761E038163F04F7B2c7a5D232492cfbfD?tab=read_proxy

Click on function `decimals` (number 6 on the list), and you'll see 18:
<img width="153" alt="image" src="https://github.com/bancorprotocol/fastlane-bot/assets/7003246/fbd557e2-8c61-4869-a748-080fcfc51555">

Reply:
OK, my bad on the fee-resolution.
I was looking into the contract's decimals function, which of course has nothing to do with the fees.
That resolution appears to be hard-coded (either in the factory contract or in the pool contract or both).
So I'm gonna assume that you've carefully investigated that in order to determine that it is indeed 4 and not 18.